tools/mo/requirements_tf.txt

```diff
@@ -1,7 +1,8 @@
 numpy>=1.16.6,<1.25.0
-tensorflow>=1.15.5,<=2.11.0
+tensorflow>=1.15.5,<=2.13.0rc0,!=2.13.0rc0
 networkx~=2.5; python_version <= "3.6"
 networkx<=2.8.8; python_version > "3.6"
 defusedxml>=0.7.1
 requests>=2.25.1
 fastjsonschema>=2.15.1,<2.17
+wrapt==1.14.1
```

Comment:
why do we need this? Please remove it

Reply:
It has been added because one of the tests started failing after upgrading TensorFlow and protobuf. It can be seen in https://dev.azure.com/openvinoci/dldt/_build/results?buildId=508669&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=cf9ee82e-e056-500f-81f4-f6a3a79d25db&l=817 (`openino-lin` job, commit `a69a21560dfc0a17afdca9041cf01aea34b01a51`).

A solution proposed in https://stackoverflow.com/questions/75583410/module-save-error-typeerror-this-dict-descriptor-does-not-support-dict recommended specifying exact wrapt version. Doing that resolved the issue.

Reply:
@p-wysocki, please put this relaxed dependency into https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/requirements_dev.txt

Reply:
I skipped the test for now and created 106651. I hope that's okay with you.